### ISSUE TITLE
Using SIFEI breaks the environment's undo function

### ISSUE THREAD

**Issue by kuleszdl** [kuleszdl/SIFEI#127]
Many actions like moving cells cannot be undone if SIFEI updates the violation markers inbetween.

**Comment by friederschueler** [comment]
Can not be fixed. No workaround known or possible!
See [this answer](https://social.msdn.microsoft.com/Forums/office/en-US/7482a0cc-33c2-45ba-84db-3e18fa3ea539/excel-undoredo?forum=exceldev) by a microsoft employee.

**Comment by kuleszdl** [comment]
I agree, but still this is an issue for the majority of SIFEI users so I'd prefer to leave this issue open.
